Shared/basic.php

```diff
@@ -272,6 +272,15 @@ function swizzleArray(&$filepost) {
 '; 
 $metadata_db->exec($sql3);
 
+$sql4 = '
+	CREATE TABLE IF NOT EXISTS gitToken ( 
+		cid INTEGER,
+		gitToken VARCHAR(50),
+		PRIMARY KEY (cid)
```

Comment:
Should this be a separate table? Maybe you could alter the existing table, as long as it does not break any existing functionality.